### ISSUE TITLE
Add a splash screen

### ISSUE THREAD

**Issue by nicosomb** [wallabag/android-app#466]
When we launch the application, I think (but I may be wrong) that it could be nice to have a splash screen with our logo (and some info like the website URL and the app version). 
Any opinion about this idea?

**Comment by Lokesh-Krishna** [comment]
This is for the first time the application is launched or for every single time that it is launched?

**Comment by di72nn** [comment]
For what purpose should we add a splash screen?

From my understanding splash screens usually when it takes some time to launch a program. It doesn't take *much* time to start the app.
If we add some info like the website URL and the app version to the splash screen, it will require us to show the splash screen for a longer time (in order to provide enough time to read all the information), which is annoying IMO.

From technical standpoint: we have a lot of stuff initialized in the `App` class which is loaded earlier than a splash screen can be shown (so a splash screen will be shown only after the internal initialization is finished), so it would require some work to implement a proper splash screen.

**Comment by storytellerr** [comment]
i want to contribute can i start?

**Comment by di72nn** [comment]
@storytellerr I advice you to choose some other issue: we're not sure about adding a splash screen.

**Comment by Lokesh-Krishna** [comment]
@di72nn A series of splash screens like in [Goodtime](https://play.google.com/store/apps/details?id=com.apps.adrcotfas.goodtime) at first launch would definitely give a more polished appearance to the app, wouldn't it?

**Comment by di72nn** [comment]
@Lokesh-Krishna can you link screenshots and/or describe the purpose of the splash screens in the Goodtime app?

So far I consider it a matter of taste. And a low-priority issue at best.

Any chance you refer to "intro screens" (I have no idea how these are called), which contain some feature descriptions, instructions and stuff? Because, I believe, these serve a different purpose and are not considered [splash screens](https://en.wikipedia.org/wiki/Splash_screen). These may be useful.

**Comment by Lokesh-Krishna** [comment]
It is definitely a low-priority issue. And yes, I think that calling them intro screens better describes their purpose. 

> which contain some feature descriptions, instructions and stuff  
This is exactly what the screens at the first launch of Goodtime do. They introduce you to the app and the concept that the app is based on (Pomodoro).

I guess what @nicosomb was talking about was [launch screens](https://material.io/guidelines/patterns/launch-screens.html#launch-screens-branded-launch) and whether or not we need it would depend entirely on how quickly the app loads. Even if it does take a while, a placeholder UI would make more sense than a branded launch screen which is what would be needed to display the logo and application and service information because I really can't remember waiting for the app to load an creating a delay just for information that is already accessible from the About screen seems unnecessary?

**Comment by di72nn** [comment]
I'm for intro screens. Among other introductory info, it may help to accustom users with offline functionality (for example, a user on Gitter recently mentioned that he wasn't aware that the app saves URLs for later uploading if there is no network available, even though the feature is available for a long time). It may also mention differences between fast/full sync types and that the fast sync should be preferred.

I'm against (not strongly, though) a splash screen. It will basically delay startup, while displaying information that is available elsewhere.

**Comment by nicosomb** [comment]
Yes yes yes, that's what I want <3

**Comment by Lokesh-Krishna** [comment]
Okay, so I came across these guidelines: [Onboarding](https://material.io/guidelines/growth-communications/onboarding.html#onboarding-top-user-benefits).

They seem to be discussing the same thing that we want to do. Ease first time users into the app. 

The approach that we have been discussing here is referred to as “Top User Benefits” approach.

This library should help in implementing the same: [Material-Onboarding](https://github.com/Vexigon/Material-Onboarding).
